### PR TITLE
report(redesign): fireworks

### PR DESCRIPTION
**Pull request by Hoten**
#8185

run this in the now deployment:
```
const scoresContainer = document.querySelector('.lh-scores-container');
scoresContainer.classList.add('score100');
document.body.classList.add('dark');
scoresContainer.addEventListener('click', _ => {
  scoresContainer.classList.toggle('fireworks-paused');
});
```

### CODE REVIEW

File: lighthouse-core/report/html/templates.html [hunk 1]
```diff
@@ -362,6 +367,7 @@
     */
     .pyro {
       display: none;
+      z-index: 1;
```
Comment (paulirish) [inline lighthouse-core/report/html/templates.html]: we need `pointer-events: none` now. it's hard but possible to click on the fireworks div blocking the click on the gauge

File: lighthouse-core/report/html/renderer/report-renderer.js [hunk 2]
```diff
@@ -244,16 +244,16 @@ class ReportRenderer {
...
-    //     const scoresContainer = this._dom.find('.lh-scores-container', headerContainer);
-    //     scoresContainer.classList.add('score100');
-    //     scoresContainer.addEventListener('click', _ => {
-    //       scoresContainer.classList.toggle('fireworks-paused');
-    //     });
-    //   }
-    // }
+    const scoresAll100 = report.reportCategories.every(cat => cat.score === 1);
+    if (!this._dom.isDevTools() && scoresAll100) {
+      if (scoresAll100) {
```
Comment (brendankenny) [inline lighthouse-core/report/html/renderer/report-renderer.js]: no need to double check `scoresAll100`

**Comment by Hoten** [comment]
> needs a test :)

ez

**Comment by paulirish** [comment]
Can we flip on dark mode for just scoresHeader?

>

**Comment by Hoten** [comment]
> Can we flip on dark mode for just scoresHeader?

i think it'd look weird dawg but @exterkamp suggests he'll try it post IO

**Comment by paulirish** [comment]
let's switch the whole report into dark mode if they are getting fireworks.

**Pull request merged**